pkg/scanners/clairify/clairify.go

```diff
@@ -412,6 +412,7 @@ func (c *clairify) GetVulnerabilities(image *storage.Image, components *scannerT
 	clairComponents := components.Clairify()
 
 	req := &clairGRPCV1.GetImageVulnerabilitiesRequest{
+		Image: utils.GetFullyQualifiedFullName(image),
```

Comment:
`FullyQualifiedFullName` is something like "http://host/path/to/image/image:tag" ?

Reply:
`GetFullyQualifiedFullName` would return something like `quay.io/repo/path:tag@sha256:digest` so `Image` probably makes sense. FWIW, changing the name of the field would change the API contract, could cause other complications / backwards compatibility issues.